src/core/feed-core.js

```diff
@@ -18,11 +18,14 @@ function getStickySqlString() {
       COALESCE(users.name, 'SYSTEM') as user_name,
       users.uuid as user_uuid,
       teams.name as team_name,
+      votes(feed_items) as votes,
```

Comment:
I think `votes` is a bit ambiguous name. It's not obviously clear if it means the number of votes, or the sum of the vote scores. Maybe it could be named vote_score?

Also, do you think there would be any performance implications for using a function here instead of inlining the counting of vote sum?